feeder/src/vote.ts

```diff
@@ -295,20 +295,34 @@ interface VoteArgs {
 }
 
 export async function vote(args: VoteArgs): Promise<void> {
-  const client = new LCDClient({
-    URL: args.lcdAddress,
-    chainID: args.chainID,
-    gasPrices: args.gasPrices,
-  })
-  const rawKey: RawKey = await initKey(args.keyPath, args.password)
-  const valAddrs = args.validator || [rawKey.valAddress]
+  let lcdAddress:string = args.lcdAddress || process.env['LCD_ADDRESS'] || ''
+  let chainID:string    = args.chainID    || process.env['CHAIN_ID']    || ''
+  let gasPrices:string  = args.gasPrices  || process.env['GAS_PRICE']   || ''
```

Comment:
This won't work for gasPrices - as the default is set in the argParser.

We'd have to move the default from the argParser to the fallback argument here.